src/index.ts

```diff
@@ -1,21 +1,29 @@
 #!/usr/bin/env node
 
+import dotenv from 'dotenv';
+
 import { createTempFolder } from './createTempFolder';
 import { fetchFile } from './fetchFile';
 import { isValidLink } from './isValidLink';
 import { showData } from './showData';
 
+dotenv.config();
+
 const downloadFilesIntoTempFolder = async (downloadLinks?: string[]) => {
-  const links: string[] = [];
+  let links: string[] = [];
   const tempFolderName = createTempFolder();
 
-  if (downloadLinks) {
-    links.push(...downloadLinks);
+  if (!downloadLinks?.length) {
```

Comment:
should be placed as the first statement to avoid unnecessary code run before and after

Reply:
done